breadcrumb.html

```diff
@@ -0,0 +1,6 @@
+<nav style="--bs-breadcrumb-divider: '>';" aria-label="breadcrumb">
```

Comment:
what does  --bs- represent?

Reply:
it represents custom property of bootstrap as bs stands for bootstrap